### PR TITLE
OKTA-172458

### PR DESCRIPTION
**Pull request by eklein-okta**
## Description:
- Various updates for User Consent

### Resolves:
[OKTA-172622](https://oktainc.atlassian.net/browse/OKTA-172622)

### CODE REVIEW

**Comment by eklein-okta** [comment]
Please let me know if this satisfies the JIRA or if I made any process errors. Thanks!  -E

File: _source/_docs/api/resources/authorization-servers.md [hunk 1]
```diff
@@ -718,7 +719,8 @@ curl -v -X POST \
      "value": "\"driving!\"",
      "conditions": {
        "scopes": [
-         "car:drive"
+         "car:drive",
+         "consent": "REQUIRED"
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/authorization-servers.md]: consent is a separate field, not inside conditions.scopes

Comment (eklein-okta) [reply _source/_docs/api/resources/authorization-servers.md]: @yuliu-okta, fixed.

File: _source/_docs/api/resources/authorization-servers.md [hunk 2]
```diff
@@ -762,7 +764,8 @@ curl -v -X PUT \
      "system": "false",
      "conditions": {
        "scopes": [
-         "car:drive"
+         "car:drive",
+         "consent": "REQUIRED",
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/authorization-servers.md]: Here also, it is not inside condition.scopes

File: _source/_docs/api/resources/oauth-clients.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Adds a new client application to your organization
 
 The [OAuth Client](#client-application-model) created by the request
 
-> {% api_lifecycle beta %} Note: Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
+> Note: Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth-clients.md]: Also in the table https://developer.okta.com/docs/api/resources/oauth-clients#client-application-properties, there are another two beta tags.

**Comment by eklein-okta** [comment]
@yuliu-okta Is this okay now?

File: _source/_docs/api/resources/authorization-servers.md (hunk 2 above)
Comment (yuliu-okta) [reply _source/_docs/api/resources/authorization-servers.md]: Here does not need to add consent, it is for claim. Instead, the missing part is the example on L575

File: _source/_docs/api/resources/oauth-clients.md (hunk 3 above)
Comment (yuliu-okta) [reply _source/_docs/api/resources/oauth-clients.md]: ping ^^

**Comment by eklein-okta** [comment]
@yuliu-okta Is this okay now?

File: _source/_docs/api/resources/authorization-servers.md [hunk 4]
```diff
@@ -716,6 +718,7 @@ curl -v -X POST \
      "claimType": "RESOURCE",
      "valueType": "EXPRESSION",
      "value": "\"driving!\"",
+     "consent": "REQUIRED",
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/authorization-servers.md]: This change is not needed.

File: _source/_docs/api/resources/authorization-servers.md [hunk 5]
```diff
@@ -762,7 +765,7 @@ curl -v -X PUT \
      "system": "false",
      "conditions": {
        "scopes": [
-         "car:drive"
+         "car:drive",
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/authorization-servers.md]: This change is not changed.

File: _source/_docs/api/resources/oauth-clients.md [hunk 6]
```diff
@@ -711,8 +711,8 @@ Client applications have the following properties:
...
 | token_endpoint_auth_method          | requested authentication method for the token endpoint. Default value: `client_secret_basic`                               | `none`, `client_secret_post`, `client_secret_basic`, or `client_secret_jwt`                  | TRUE     | FALSE  | FALSE     |
 | initiate_login_uri                  | URL that a third party can use to initiate a login by the client                                                           | String                                                                                       | TRUE     | FALSE  | FALSE     |
-| tos_uri {% api_lifecycle beta %}    | URL string of a web page providing the client's terms of service document                                                                                         | URL                                                                                          | TRUE     | FALSE  | FALSE     |
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth-clients.md]: We need the EA tag for these two properties.

File: _source/_docs/api/resources/oauth-clients.md [hunk 7]
```diff
@@ -40,7 +40,7 @@ Adds a new client application to your organization
 
 The [OAuth Client](#client-application-model) created by the request
 
-> {% api_lifecycle beta %} Note: Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth-clients.md]: EA tag

**Comment by eklein-okta** [comment]
@yuliu-okta Trying again... :)

File: _source/_docs/api/resources/authorization-servers.md [hunk 8]
```diff
@@ -762,7 +764,6 @@ curl -v -X PUT \
      "system": "false",
      "conditions": {
        "scopes": [
-         "car:drive"
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/authorization-servers.md]: Sorry, I mean the change, adding a comma, is not needed, this original content is still valid. This line should keep as `"car:drive"`

Comment (eklein-okta) [reply _source/_docs/api/resources/authorization-servers.md]: Done.

**Pull request merged**